otelx/config.schema.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "ory://tracing-config",
+  "type": "object",
+  "additionalProperties": false,
+  "description": "Configure distributed tracing using OpenTelemetry",
+  "properties": {
+    "provider": {
+      "type": "string",
+      "description": "Set this to the tracing backend you wish to use. Supports Jaeger.",
+      "enum": [
+        "jaeger"
+      ],
+      "examples": [
+        "jaeger"
+      ]
+    },
+    "service_name": {
+      "type": "string",
+      "description": "Specifies the service name to use on the tracer.",
+      "examples": [
+        "Ory Hydra",
+        "Ory Kratos",
+        "Ory Keto",
+        "Ory Oathkeeper"
+      ]
+    },
+    "providers": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "jaeger": {
+          "type": "object",
+          "additionalProperties": false,
+          "description": "Configures the jaeger tracing backend.",
+          "properties": {
+            "local_agent_address": {
+              "type": "string",
+              "description": "The address of the jaeger-agent where spans should be sent to.",
+              "oneOf": [
+                {
+                  "pattern": "^\\[(([0-9a-fA-F]{1,4}:){7}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,7}:|([0-9a-fA-F]{1,4}:){1,6}:[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,5}(:[0-9a-fA-F]{1,4}){1,2}|([0-9a-fA-F]{1,4}:){1,4}(:[0-9a-fA-F]{1,4}){1,3}|([0-9a-fA-F]{1,4}:){1,3}(:[0-9a-fA-F]{1,4}){1,4}|([0-9a-fA-F]{1,4}:){1,2}(:[0-9a-fA-F]{1,4}){1,5}|[0-9a-fA-F]{1,4}:((:[0-9a-fA-F]{1,4}){1,6})|:((:[0-9a-fA-F]{1,4}){1,7}|:)|fe80:(:[0-9a-fA-F]{0,4}){0,4}%[0-9a-zA-Z]{1,}|::(ffff(:0{1,4}){0,1}:){0,1}((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])|([0-9a-fA-F]{1,4}:){1,4}:((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9]))]:([0-9]*)$"
+                },
+                {
+                  "pattern": "^([0-9]{1,3}\\.){3}[0-9]{1,3}:([0-9]*)$"
+                },
+                {
+                  "format": "uri"
```

Comment:
Format `uri` requires a scheme, but the code does not allow for parsing schemes, only `hostname:port` combinations.

Reply:
Yeah I get that, but removing `"format": "urI"` causes tests to fail and I'm not sure why.